testcontainers/compose.py

```diff
@@ -88,7 +90,7 @@ def start(self):
             pull_cmd = self.docker_compose_command() + ['pull']
             subprocess.call(pull_cmd, cwd=self.filepath)
 
-        up_cmd = self.docker_compose_command() + ['up', '-d']
+        up_cmd = self.docker_compose_command() + ['up', '-d'] + ["--build"] if self.build else []
```

Comment:
I think the tests may be failing because of the operator precedence here (the ternary has higher precedence than the addition). Let's change this to:

```suggestion
        up_cmd = self.docker_compose_command() + ['up', '-d']
        if self.build:
            up_cmd.append('--build')
```

Reply:
Fixed.